lib/api/src/grpc/models.rs

```diff
@@ -4,17 +4,20 @@ use schemars::JsonSchema;
 use serde;
 use serde::{Deserialize, Serialize};
 
-#[derive(Serialize, Deserialize)]
+#[derive(Serialize, Deserialize, JsonSchema)]
 pub struct VersionInfo {
     pub title: String,
     pub version: String,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub commit_id: Option<String>,
```

Comment:
Nitpick: I believe this is commonly called `commit`, would that be better?

Reply:
I was thinking the same. Will do. Thanks!